docs/migration/checklist.md

```diff
@@ -0,0 +1,26 @@
+# Migration Checklist
+
+### The Source Repo has 
+  - [ ] Dockerfile
+  - [ ] build_deploy.sh
+  - [ ] pr_check.sh
+  - [ ] e2e build disabled (after build pipeline is validated)
+### App config (check if not applicable)
+  - [ ] Kafka topics and bootstrap url
+  - [ ] Object store buckets (minio, s3)
+  - [ ] Databases (RDS, Redis)
+  - [ ] Logging via Cloudwatch
+  - [ ] Web path and port 
+  - [ ] Metrics path and port  
+  - [ ] Dependent services (rbac, etc) 
+### Clowdapp resource
```

Comment:
We should add all the things not related to pod spec, e.g. kafka topics, web and in-memory db flags, etc.